### DISCUSSION TITLE
How to see all connections that were denied w/o GUI interaction?

### DISCUSSION THREAD

**Discussion by BetweenTheKeyboardAndChair** [evilsocket/opensnitch#1109] [Q&A]
Is there a way to search the logs to see which connections got denied without a rule or GUI interaction? 
(Actually anything that is "deny" should be fine.  I never deny by rule; I always allow or reject.)
A copy/paste command would be appreciated.

I suspect that the GUI isn't getting notified of all of the changes when there are a lot of connections flying by (99% of which already have rules).  My "evidence" is that if I set the default for no-GUI to "allow", things run smoothly.  If I set the default to "deny" when no-GUI, I get hangs.  I'm responding to every GUI pop up.

Linux ucin 5.4.0-172-generic #190-Ubuntu
I get the epbf warning.  (Should I be changing the setting to proc?  I forget   (I upgraded opensnitch recently.)

**Comment by gustavo-iniguez-goya** [comment]
You can filter denied connections with ... `tail -f /var/log/opensnitchd.log | grep ✘` 

Could you describe the odd behaviour you're experiencing? to have an idea of what the symptoms are.

> I get the epbf warning. (Should I be changing the setting to proc? I forget (I upgraded opensnitch recently.)

Could you post the ebpf warning? mmh, you're running kernel 5.4, so maybe the opensnitch-procs module failed to load. But it shouldn't interefere with the connections interception.

Enable `[x] Debug invalid connections` under Preferences->Nodes, and see if you're prompted to allow more outbound connections.
